### ISSUE TITLE
Broken design on mobile/tablet

### ISSUE THREAD

**Issue by sboldyreva** [AlmaLinux/almalinux.org#610]
After some recent changes, the website design looks broken for different screen sizes including the community section. @bennyvasquez @codyro @jonathanspw 
This has to be adapted and fixed:
![image](https://github.com/user-attachments/assets/c01ae4f7-28e5-4531-9c63-613986b994df)
![image](https://github.com/user-attachments/assets/7ea6cc24-92f7-4840-b10e-1140f413a461)
![image](https://github.com/user-attachments/assets/bec5c8d7-4744-4315-91e9-851d508bbf8f)
![image](https://github.com/user-attachments/assets/afd66ecf-cd0f-497c-a8c2-14e0d9280a00)
![image](https://github.com/user-attachments/assets/25646d3f-f65c-4cbf-9a70-111155978e70)
![image](https://github.com/user-attachments/assets/63b0fef7-d1c5-4da7-8a45-f9f4c84338f1)

**Comment by bennyvasquez** [comment]
Yes, please! <3

**Comment by codyro** [comment]
That's pretty bad. It might be worth reverting that change until we can fix it up.

**Comment by bennyvasquez** [comment]
Definitely not reverting the changes, so we’ll need to prioritize the fixes. @Shone-Kristas or @mattrandomnumber: any chance you’ve got some space to take a look this weekend?

**Comment by codyro** [comment]
Out or curiosity, why? It's a bad look for AlmaLinux and sponsors that the core website is busted on a lot of resolutions. If you're visiting an EL distribution website, and it's completely broken when you see it, it's not really instilling confidence. 

I don't agree with this. It seems short sighted. This absolutely should be reverted until it's fixed. Otherwise we have a completely broken website for no reason that I can discern. 

tldr; at no company or project I've been involved in would this be an acceptable way to approach something broken in production that can be resolved immediately until a proper fix is in place.

**Comment by bennyvasquez** [comment]
According to our webstats less than a tenth of our website traffic is on mobile devices, and "completely broken" is a bit hyperbolic. 

If we can't resolve it this weekend, I'll submit a Pr on Monday reverting the newsletter addition and the new commercial support displays.

**Comment by sboldyreva** [comment]
https://github.com/AlmaLinux/almalinux.org/pull/612
